packages/edit-post/src/editor.native.js

```diff
@@ -26,6 +26,10 @@ class Editor extends Component {
 	constructor( props ) {
 		super( ...arguments );
 
+		if ( ! __DEV__ && this.props.capabilities.stories !== true ) {
+			this.props.hideBlockTypes( [ 'jetpack/story' ] );
+		}
```

Comment:
I think we are falling in the same problem than before:
This repo should know nothing about the stories block, nor mention jetpack in any way.

We should do this either on `gutenberg-mobile` or `jetpack` repos.

Reply:
ouch, keep forgetting about this. Thanks for the heads up @etoledom ! Will try moving it elsewhere 👍

Reply:
@etoledom moved things out of Guteberg in this commit 520d6491d0, and implemented the logic in gutenberg-mobile in this commit https://github.com/wordpress-mobile/gutenberg-mobile/pull/2762/commits/a361b1d6328ce0e044d11da32f80acd923947bdf

Also renamed the `stories` capability flag to convey the more general abstraction as done in the previous refactors (changes `stories` capability name for `mediaFilesCollectionBlock`).

Ready to for another pass 🙇